db/seed_data/districts.sql

```diff
@@ -0,0 +1,441 @@
+INSERT INTO districts (id, district, state_id, created_at, updated_at) VALUES (31, '17', 6, '2015-02-06 00:48:06.415016', '2015-02-06 00:48:06.416166');
+INSERT INTO districts (id, district, state_id, created_at, updated_at) VALUES (32, '18', 6, '2015-02-06 00:48:06.421442', '2015-02-06 00:48:06.42334');
+INSERT INTO districts (id, district, state_id, created_at, updated_at) VALUES (35, '20', 6, '2015-02-06 00:48:06.435113', '2015-02-06 00:48:06.436709');
```

Comment:
We're inserting state_id, district_id, and zip_code_id into other tables, so it's essential that they match the id value from the db on my system that the data was copied from. We could could skip id field on legislators, though